### PR TITLE
Db config

### PR DESCRIPTION
**Pull request by maximderbin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,8 +29,7 @@ carpool
 #### Setting up postgres
 1. brew install postgres
 2. npm install
-3. postgres -D /usr/local/var/postgres
-4. In psql, `create database carpool_dev;`
+3. `postgres -D /usr/local/var/postgresq
```
Comment (scsper) [inline README.md]: s/postgresq/postgres

File: README.md [hunk 2]
```diff
@@ -8,18 +8,16 @@ carpool
 2. Add the following line to `/etc/hosts/`: `127.0.0.1       localhost       local.carpool.com`
 3. Set up postgres (see below)
 4. Start postgres `postgres -D /usr/local/var/postgres`
-5. While postgres is running, perform migrations `gulp migrate`
-6. `npm install -g gulp`
-7. `npm install -g gulp-cli`
-8. Open new shell and start the web server: `gulp`
-9. Go to http://local.carpool.com:3000
+5. Bootsrap application `make bootstrap`
```
Comment (scsper) [inline README.md]: s/Bootsrap/Bootstrap

**Comment by scsper** [comment]
@maximderbin Thanks for the PR!  Could you also add a line to the README that specifies how developers can set up the config file?  It looks like it's just `make db.config`, but I'd like to make sure before merging :)

**Comment by scsper** [comment]
@dmitriiabramov What do you think about moving the Makefile into Gulp?  That's something I would like done in a subsequent PR, unless there's a good reason not to.   I'd like to stick with one task runner.

**Comment by maximderbin** [comment]
@scsper Okay will move it to `Gulp`. Thought about that during implementation but skipped for some reason. And also will list all commands in readme.

**Comment by scsper** [comment]
Thanks @maximderbin!

**Comment by maximderbin** [comment]
@scsper updated, any comments on that?

**Comment by maximderbin** [comment]
@dmitriiabramov @scsper updated, any comments on that?

**Comment by scsper** [comment]
:+1:

**Pull request merged**